### PR TITLE
Report - Checking accessibility for my portfolio.

### PR DESCRIPTION
**Pull request by diegoh40**
1 - Page titles - No changes required.
2 - Image text alternatives - changed.
[Changed  'icon' to 'menu' ](https://github.com/diegoh40/Diego-Portfolio/blob/8e1455d3957f05f28f778cbf425e8876d955518c/index.html#L29)
[Changed 'icon' to 'email'](https://github.com/diegoh40/Diego-Portfolio/blob/8e1455d3957f05f28f778cbf425e8876d955518c/index.html#L32)
[Changed 'social ' to 'specific social'](https://github.com/diegoh40/Diego-Portfolio/blob/6e58a6998c2592e92280746b90851e584270baa1/index.html#L53-L65)
[Changed 'works first images ' to 'works first image'](https://github.com/diegoh40/Diego-Portfolio/blob/6e58a6998c2592e92280746b90851e584270baa1/index.html#L83)
[Changed removing text inside alt](https://github.com/diegoh40/Diego-Portfolio/blob/6e58a6998c2592e92280746b90851e584270baa1/index.html#L297)
[Changed removing text inside alt](https://github.com/diegoh40/Diego-Portfolio/blob/6e58a6998c2592e92280746b90851e584270baa1/index.html#L310)
[Changed removing text inside alt](https://github.com/diegoh40/Diego-Portfolio/blob/6e58a6998c2592e92280746b90851e584270baa1/index.html#L323)
[Changed 'social ' to 'specific social'](https://github.com/diegoh40/Diego-Portfolio/blob/6e58a6998c2592e92280746b90851e584270baa1/index.html#L385-L397)

3 - Text headings - No changes required.
4 - Color contrast - No changes required.
5 - Resize - No changes required.
6 - Interactions - No changes required.
7 - Moving content - No changes required.
8 - Multimedia - No changes required.
9 - Basic page structure - No changes required.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -95,8 +96,12 @@ <h3>Multi-Post Stories</h3>
           <li>boostrap</li>
           <li>ruby</li>
         </ul>        
-        <!-- <div class="button_works"> -->
-          <form class="button_works " action="https://github.com/diegoh40">
+        
+          <!-- <form class="button_works " action="https://github.com/diegoh40">
+            <button type="submit">See Project</button>
+          </form> -->
```
Comment (youmari) [inline index.html:102]: - [ ] please try to remove unused commented code in order to keep your file professional

**Review by youmari — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights

Git/GitHub flow best practices have been followed✔️
There are no linter errors✔️
You have descriptive PR✔️

## Required Changes ♻️

- [ ] it would be better if you use the `<label>` tag for all From control inside the `<form>` tag in your contact section in order for the google bot can understand your website what  is about
improve your SEO benchmark

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by diegoh40** [comment]
I have made change suggestions to de CR to let this code be clear and proper.

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @diegoh40 ad Team👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good accessibility report. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**